src/makefile.mk

```diff
@@ -39,7 +39,9 @@ CUSTOM_FILE_FILE ?= stdio_file.h
 DEPS ?=
 #----------------------------
 HAS_UPPERCASE_NAME ?= YES
-HAS_FLASH_FUNCTIONS ?= YES
+# Prefer static crt functions.
+# These will increase output size but are usually faster than those from TI-OS.
+STATIC_CRT ?= YES
```

Comment:
Do people want a different name/behavior for this?

Reply:
Seems fine to me